custodia/secrets.py

```diff
@@ -259,6 +259,8 @@ def _int_get_key(self, trail, request, response):
             output = self.root.store.get(key)
             if output is None:
                 raise HTTPError(404)
+            elif len(output) == 0:
+                raise HTTPError(406)
```

Comment:
I think it breaks the case of an empty string as secret value, too. An application can store ```''```` in Custodia.